docs/workflows.md

```diff
@@ -231,14 +274,16 @@ If you use the "static data" field (pencil), you have to write it out yourself.
 
 PS: it still only works with PREVIOUS nodes (nodes before the one you're editing)
 
-### Parsing JSON
+## Parsing JSON
 Parsing JSON is essential to be able to get API data. Shuffle uses it's own schema, which seems to work quite well. 
 
-$ 					= identifier for a previous node / execution argument 
```

Comment:
I found this example ambigious when first looking at the docs. Initially, I thought a literal  single`$` by itself (or in the form of `$.results`) was referring to the previous node (IE, if the workflow had nodes A->B->C, if used in C would be referring only explicitly to B. Hence, I thought it might be easier to understand to use an example in the form of `$node_name`

Reply:
Actually, perhaps `$shuffle_tools_1` or `$change_me` is a clearer example to use here. As this will be very obvious where the linkage is to that being the node's name with spaces substituted for underscores. Though, is that mentioned anywhere currently, either?? 🤔

Reply:
Good point! We've had multiple people make that mistake some time back, so it may indeed be better. Gonna read over the changes a bit better and merge soon :)